bin/cask

```diff
@@ -254,7 +254,9 @@ def get_cask_emacs():
     # compile.  In this case, $EMACS may be set as well to a rather
     # meaningless value.  If $INSIDE_EMACS is set, we just ignore the
     # value of $EMACS.
+    #print( ENVB )
     emacs = ENVB.get(b'EMACS') if b'INSIDE_EMACS' not in ENVB else None
+    emacs = ENVB.get(b'CASK_EMACS') or emacs
```

Comment:
Please change this code to check this variable first, before the other one.  It's strange to check a variable that's supposed to take precedence at last.  

Please also change this code to not require three assignments to `emacs`.  I guess you can move the environment variable access into a separate function, so that the line below just reads `emacs = get_emacs_from_env() or find_best_emacs()`.